share/smack/top.py

```diff
@@ -420,13 +420,21 @@ def verify_bpl(args):
     command += ["/maxStaticLoopBound:%d" % args.loop_limit]
     command += ["/recursionBound:%d" % args.unroll]
 
+  elif args.verifier == 'symbooglix':
+    command = ['symbooglix']
+    command += [args.bpl_file]
+    command += ["--file-logging=0"]
+    command += ["--entry-points=%s" % ",".join(args.entry_points)]
+    command += ["--timeout=%d" % args.time_limit]
+    command += ["--max-depth=%d" % args.unroll]
```

Comment:
Yes, this is the behavior I would expect.
Now, I remember that either Boogie or Corral have a different (and less intuitive) behavior when it comes to nested loops. But what you suggested makes sense.